crosswalks/Dublin Core.csv

```diff
@@ -1,59 +1,59 @@
 Property,Dublin Core
-codeRepository,
-programmingLanguage,
-review,
+codeRepository,source
+programmingLanguage,publisher
+review,relation
 runtimePlatform,
 targetProduct,
-applicationCategory,
-applicationSubCategory,
-downloadUrl,
+applicationCategory,subject
+applicationSubCategory,subject
+downloadUrl,identifier
 fileSize,
 installUrl,
 memoryRequirements,
 operatingSystem,
-permissions,
+permissions,accessRights
 processorRequirements,
-releaseNotes,
+releaseNotes,description
 softwareHelp,
-softwareRequirements,
-softwareVersion,
-storageRequirements,
-supportingData,
+softwareRequirements,requires
+softwareVersion,hasVersion
+softwareSuggestions,
+supportingData,relation
 author,creator
-citation,
-contributor,
-copyrightHolder,
-copyrightYear,
+citation,bibliographicCitation
+contributor,contributor
+copyrightHolder,rightsHolder
+copyrightYear,dateCopyrighted
 dateCreated,created
-dateModified,
-datePublished,date
-editor,
-encoding,
-fileFormat,
-funder,
-keywords,
+dateModified,modified
+datePublished,issued
+editor,contributor
+encoding,format
+fileFormat,format
+funder,creator
+keywords,subject
 license,license
-producer,
+producer,contributor
 provider,
 publisher,publisher
-sponsor,
-version,dcterms:hasVersion
-isAccessibleForFree,
-isPartOf,
-hasPart,
+sponsor,contributor
+version,hasVersion
+isAccessibleForFree,accessRights
+isPartOf,isPartOf
+hasPart,hasPart
 position,
 description,description
 identifier,identifier
 name,title
-sameAs,
-url,
-relatedLink,
+sameAs,relation
+url,identifier

```

Comment:
Hmmmm I don't like that we mix identifier and URL, but I don't have a better option to suggestion.